### DISCUSSION TITLE
文献\cite{没有自动补全提示

### DISCUSSION THREAD

**Discussion by yedvvard** [note286/xduts#157] [Q&A] [answered]
使用vscode+latex-workshop，输入\cite{后没有自动补全提示，查看latex-workshop文档发现它是靠解析其中的bib相关语句来解析bib相关的信息的，xduts包中好像将\addbibresources封装的很好，latex-workshop不能找到使用的bib文件，在我的机器上即使在latex-workshop的配置中设置`"latex-workshop.latex.bibDirs"`也没能正确显示提示，最终的解决办法是使用一个注释来实现，在main.tex中：
```
\begin{document}

\iffalse 
\addbibresource{path-to-bib.bib}
\fi 

\include{chapters/chapter1_preliminary}
\include{chapters/chapter2_bases}
\include{chapters/chapter3_work1}
\end{document}
```

这样就能正常显示提示了

**Comment by note286** [comment, marked as answer]
你做的对，我开发的时候没想到这个问题，后来想到了，但是没啥时间处理。关于xduts攒了一堆想法，等后面有时间了一定处理。

**Reply by Chang-007** [reply]
感谢解答！已解决

**Comment by wanstarge** [comment]
原来在这,我还在找插件的问题
